client-templates/artifactory/.env.sample

```diff
@@ -13,3 +14,6 @@ ACCEPT=accept.json
 
 # The path for the broker's internal healthcheck URL. Must start with a '/'.
 BROKER_HEALTHCHECK_PATH=/healthcheck
+
+# Provide RES_BODY_URL_SUB with the URL of the artifactory without credentials and http protocol
```

Comment:
Not sure that description is clear. Can you elaborate a bit more about `RES_BODY_URL_SUB `? Why we need it?

Reply:
Why we need it - when npm does an install it will first grab metadata about the package. This metadata includes where the tarball for each version is located. The url location for the tarball will be the artifactory url, which is inaccessible to us, and needs to be substituted to the broker url base.

Reply:
Should this variable be renamed or only the description?

Reply:
@orsagie Thanks for clarifying this☺️ I think it might be a good idea to update description. Don't have any suggestions on naming🤷‍♂️

Reply:
updated the description